sdk/include/reactos/wine/winternl.h

```diff
@@ -1813,7 +3401,7 @@ typedef struct _RTL_HANDLE_TABLE
 #define FILE_OPEN_FOR_BACKUP_INTENT     0x00004000
 #define FILE_NO_COMPRESSION             0x00008000
 #define FILE_RESERVE_OPFILTER           0x00100000
-#define FILE_TRANSACTED_MODE            0x00200000
+#define FILE_OPEN_REPARSE_POINT         0x00200000
```

Comment:
Did this flag change meaning or is it just renamed?
If it changed, that might be a newer windows thing? (7+ or 10+?)